tools/datagen/src/bin/datagen.rs

```diff
@@ -380,6 +380,41 @@ fn main() -> eyre::Result<()> {
         }
     })?;
 
+    // Segmenter's rule data depends on various Unicode Properties. We have to generate them after
+    // uprops.
+    if matches.is_present("TEST_KEYS") {
+        // FIXME: We should support specifying segmenter keys in --keys.
```

Comment:
Please fix

Reply:
Done in [9bfca00](https://github.com/unicode-org/icu4x/pull/1652/commits/9bfca007afd38cfe47e46b048c0de2d2e44671e2)